### DISCUSSION TITLE
Option in Windows 11 for 'Legacy' Start Button Behaviour – MouseDown/Click to Select

### DISCUSSION THREAD

**Discussion by Doctor-of-Music** [Open-Shell/Open-Shell-Menu#1983] [Ideas]
On the Start Button in Windows versions up to 10, it was possible to use left mouse button action Press (mousedown) – scroll and select program – Release (mouseup). In Windows 11 this no longer works – running a program requires Click (mousedown-mouseup) – scroll to select – Click a second time.
Would it be possible to add an option to restore 'legacy' Start Button behaviour to Open Shell on Windows 11 please?

Otherwise (& the above is not an Open Shell bug but a Windows issue) Open Shell is working very nicely for me in Win11 LTSC (24H2) using the Classic Menu option. I restored the older file explorer (without tabs) which is apparently 'hidden' inside the new Explorer using the registry patch described here:
https://www.neowin.net/guides/how-to-permanently-restore-old-file-explorer-in-windows-11/
Some of the Classic Explorer settings seem to work with the 'hidden' legacy Explorer.

**Comment by ge0rdi** [comment]
> On the Start Button in Windows versions up to 10, it was possible to use left mouse button action Press (mousedown) – scroll and select program – Release (mouseup).

Just to clarify, do you mean behavior of original Windows Start button or `Open-Shell` start button?
Because original button doesn't seem to work that way (menu is displayed on left button release).

`Open-Shell` start menu seems to be working the way you described.
For me it works that way even with latest Win11 Insider.
